### ISSUE TITLE
Get a data review on metrics

### ISSUE THREAD

**Issue by wresuolc** [mozilla/FirefoxColor#253]
This depends on #11 though

https://github.com/mozilla/FirefoxColor/blob/master/docs/metrics.md

**Comment by wresuolc** [comment]
1. Is there documentation that describes the schema for the ultimate data set available publicly, complete and accurate?
    Yes, see link above to the metrics doc

2.   Is there a control mechanism that allows the user to turn the data collection on and off?
    Installing/Uninstalling the add-on
 
3. What type of data do the requested instruments or measurements fall under?
    category 1 and 2  (let's verify this after #11 is closed)

4. Is the data collection default-on or default-off?
    Default-on for all users in the Test Pilot trial.

5.  Does the instrumentation include the addition of any new identifiers (whether anonymous or otherwise; e.g., username, random IDs, etc. See the appendix for more details)?
    No. As with all Test Pilot trials, there is a UUID to allow for tracking events throughout the event measurement sequence and a Google Analytics property ID. Neither of these are new to this proposal, however.

6. Is the data collection covered by the existing Firefox privacy notice?
    Yes

7. Does there need to be a check-in in the future to determine whether to renew the data?
    If the results of this Test Pilot study result in a decision to ship the feature to release, yes. Otherwise, no.

**Comment by wresuolc** [comment]
@rjweiss -- please review our data collection at https://github.com/mozilla/FirefoxColor/blob/master/docs/metrics.md

**Comment by rayborn** [comment]
r+

# Data Review Form

1) Is there or will there be **documentation** that describes the schema for the ultimate data set available publicly, complete and accurate?
Yes, metrics.md

2) Is there a control mechanism that allows the user to turn the data collection on and off? (Note, for data collection not needed for security purposes, Mozilla provides such a control mechanism) Provide details as to the control mechanism available.
Yes, enrollment in Test Pilot

3) If the request is for permanent data collection, is there someone who will monitor the data over time?
Non-permanent due to the nature of Test Pilot, Test Pilot team on analysis for the lifetime of the extension.

4) Using the **[category system of data types](https://wiki.mozilla.org/Firefox/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?
Category 2 and below

5) Is the data collection request for default-on or default-off?
default-on under the Test Pilot program

6) Does the instrumentation include the addition of **any *new* identifiers** (whether anonymous or otherwise; e.g., username, random IDs, etc.  See the appendix for more details)?
No

7) Is the data collection covered by the existing Firefox privacy notice? 
Yes, Test Pilot policies and general Mozilla policies

8) Does there need to be a check-in in the future to determine whether to renew the data?
No

**Comment by wresuolc** [comment]
thank you
